src/main/java/ro/societateahermes/backendservice/service/serviceImplementation/UserServiceImplementation.java

```diff
@@ -1,30 +1,57 @@
 package ro.societateahermes.backendservice.service.serviceImplementation;
 
+import org.modelmapper.ModelMapper;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Service;
+import ro.societateahermes.backendservice.entities.dto.MySubmissionDTO;
+import ro.societateahermes.backendservice.entities.dto.UserDTO;
+import ro.societateahermes.backendservice.entities.Participation;
 import ro.societateahermes.backendservice.entities.User;
 import ro.societateahermes.backendservice.repository.UserRepositoryInterface;
 import ro.societateahermes.backendservice.service.UserServiceInterface;
 
 import java.util.List;
+import java.util.stream.Collectors;
 
 @Service
 public class UserServiceImplementation implements UserServiceInterface {
-    private final UserRepositoryInterface userRepository;
-    /* private final ActivityRepositoryInterface activityRepository;
-    private final EventRepositoryInterface eventRepository;*/
 
-    UserServiceImplementation(UserRepositoryInterface userRepo) {
-        userRepository = userRepo;
+    @Autowired
+    private UserRepositoryInterface userRepository;
+
+    @Autowired
+    private ModelMapper modelMapper;
+
+
+
+    @Override
+    public User saveUserFromDTO(MySubmissionDTO submissionDTO) {
+
+        User user = new User();
+
+        user.setFirstName(submissionDTO.getFirstName());
+        user.setLastName(submissionDTO.getLastName());
+        user.setEmail(submissionDTO.getEmail());
+        user.setUsername(submissionDTO.getUsername());
+        user.setPassword(submissionDTO.getPassword());
+        user.setUniversity(submissionDTO.getUniversity());
```

Comment:
Operatiile astea in care construiesti noul user ar trebui sa fie facute nu in service, ci intr-un mapper. Daca faci asa atunci service-ul capata responsabilitatea de a converti din dto in entitate si invers, ceea ce nu-i bine